check-diff.sh

```diff
@@ -494,12 +494,18 @@ function run_phpunit_local {
 				cd - > /dev/null
 			fi
 
-			if [ ! -z "$ABSOLUTE_VAGRANT_PATH" ]; then
+            if [ ! -z "$ABSOLUTE_VAGRANT_PATH" ]; then
 				echo "Running phpunit in Vagrant"
-				vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && phpunit -c $PHPUNIT_CONFIG"
+				if [ -n "$PHPUNIT_CONFIG" ]; then
+				    vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && phpunit -c $( echo "$PHPUNIT_CONFIG"; )"
+				else
+				    vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && find . -iname phpunit.xml.dist -execdir phpunit \;"
```

Comment:
Basically I think the the command passed into `vagrant ssh` should be some form of:

```bash
if [ -n "$PHPUNIT_CONFIG" ] || [ -e phpunit.xml* ]; then
	phpunit $( if [ -n "$PHPUNIT_CONFIG" ]; then echo -c "$PHPUNIT_CONFIG"; fi )
fi
for nested_project in $( find $PATH_INCLUDES -name 'phpunit.xml*' -mindepth 2 ! -path '*/vendor/*' -name 'phpunit.xml*' -exec dirname {} \; ); do
	(
		cd "$nested_project"
		phpunit
	)
done
```

It would be ideal of this logic wasn't duplicated in multiple places, but a refactor would be complicated.